src/apiFunctions.js

```diff
@@ -2,24 +2,29 @@ function generateAPIUrl(baseId, tableName) {
   return `https://api.airtable.com/v0/${baseId}/${tableName}`;
```

Comment:
Good intuition in creating your function with the option to work on different tables. You could have hard-coded this which would work fine but you no longer need to refactor this if you add additional tables.